e2e_tests/integration/0.index.spec.ts

```diff
@@ -45,13 +46,67 @@ describe('Neo4j Browser', () => {
       .contains(':server connect')
       .click()
     cy.typeInFrame(':play movies{enter}', 0)
-    cy.get('[data-testid=frame]').contains('the Bacon Path')
+    cy.get('[data-testid="frame"]').contains('the Bacon Path')
   })
+
   it('can connect', () => {
     const password = Cypress.config('password')
     cy.connect('neo4j', password)
   })
 
+  it('can show Canny badge in "Document" icon and correspond side bar drawer', () => {
+    cy.window().then(win => {
+      if (win.Canny && win.IsCannyLoaded && typeof win.Canny === 'function') {
```

Comment:
Good to not fail any releases if the canny sdk were to not load properly due to network issues or simliar